.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
   # Run all tests before db_block_cache_test (db_test, db_test2)
   - JOB_NAME=unittests ROCKSDBTESTS_END=db_block_cache_test
   # Run all tests starting from db_block_cache_test (db_block_cache_test, ..., plain_table_db_test)
-  - JOB_NAME=unittests ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
+  - JOB_NAME=unittests MEM_ENV=true ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
```

Comment:
Need to make sure external_sst_file_test, db_io_failure_test, checkpoint_test and corruption_test don't run on Mem Env. Maybe you've already done so. I didn't yet check the detail. column_family_test probably shouldn't go to MemEnv either.